.gitignore

```diff
@@ -131,3 +131,5 @@ dist/
 
 /media_dir.txt
 # ^TODO: Remove the need for this with a proper config file
+.coverage.*
+.coverage
```

Comment:
what's the reasoning adding these?

Reply:
When I ran pytest, it generated a lot of test coverage files.

Reply:
Unless you accidentally `git add`ed them then they shouldn't have been included in the PR. Maybe `git rm` on those files.

Reply:
I didn't add them. Since it was my first PR I wanted to be extra careful, so I added them to the gitignore as well.

Reply:
I think it's fine to remove these lines from the `.gitignore` in that case